<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="openFileAction"/>
    <addaction name="saveFileAction"/>
    <addaction name="saveFileAsAction"/>
    <addaction name="exportHtmlAction"/>
    <addaction name="closeAction"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="previewModeAction"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="boldFmtAction"/>
    <addaction name="italicFmtAction"/>
    <addaction name="codeFmtAction"/>
    <addaction name="ulistFmtAction"/>
    <addaction name="olistFmtAction"/>
    <addaction name="linkFmtAction"/>
    <addaction name="imageFmtAction"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuView"/>
   <addaction name="menuFormat"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openFileAction"/>
   <addaction name="saveFileAction"/>
   <addaction name="saveFileAsAction"/>
   <addaction name="separator"/>
   <addaction name="previewModeAction"/>
   <addaction name="separator"/>
   <addaction name="boldFmtAction"/>
   <addaction name="italicFmtAction"/>
   <addaction name="ulistFmtAction"/>
   <addaction name="olistFmtAction"/>
   <addaction name="codeFmtAction"/>
   <addaction name="linkFmtAction"/>
   <addaction name="imageFmtAction"/>
  </widget>
  <action name="openFileAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveFileAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="saveFileAsAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/save-as.png</normaloff>:/icons/save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="exportHtmlAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/web.png</normaloff>:/icons/web.png</iconset>
   </property>
   <property name="text">
    <string>Export Html</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/quit.png</normaloff>:/icons/quit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="previewModeAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/preview.png</normaloff>:/icons/preview.png</iconset>
   </property>
   <property name="text">
    <string>Preview Mode</string>
   </property>
  </action>
  <action name="boldFmtAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/bold.png</normaloff>:/icons/bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="italicFmtAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/italic.png</normaloff>:/icons/italic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="codeFmtAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/code.png</normaloff>:/icons/code.png</iconset>
   </property>
   <property name="text">
    <string>Inline Code</string>
   </property>
  </action>
  <action name="ulistFmtAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/bullet.png</normaloff>:/icons/bullet.png</iconset>
   </property>
   <property name="text">
    <string>Unordered List</string>
   </property>
  </action>
  <action name="olistFmtAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/number.png</normaloff>:/icons/number.png</iconset>
   </property>
   <property name="text">
    <string>Ordered List</string>
   </property>
  </action>
  <action name="linkFmtAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/link.png</normaloff>:/icons/link.png</iconset>
   </property>
   <property name="text">
    <string>Insert Link</string>
   </property>
  </action>
  <action name="imageFmtAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/image.png</normaloff>:/icons/image.png</iconset>
   </property>
   <property name="text">
    <string>Insert Image</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
